app/src/main/res/xml/preferences.xml

```diff
@@ -121,21 +121,14 @@
     <PreferenceCategory
         android:key="pref_live_tv_category"
         android:title="@string/pref_live_tv_cat">
-        <CheckBoxPreference
-            android:defaultValue="false"
-            android:disableDependentsState="true"
-            android:key="pref_live_tv_use_external"
-            android:summary="@string/pref_external_live_tv_summary"
-            android:title="@string/pref_external_live_tv_title" />
-        <CheckBoxPreference
-            android:defaultValue="false"
-            android:dependency="pref_live_tv_use_external"
-            android:key="pref_enable_vlc_livetv"
-            android:summary="@string/desc_use_vlc_livetv"
-            android:title="@string/lbl_use_vlc_livetv" />
+        <ListPreference
+            android:entries="@array/pref_video_player_entries"
+            android:entryValues="@array/pref_video_player_values"
+            android:key="live_tv_video_player"
+            android:title="@string/pref_media_player"
+            app:useSimpleSummaryProvider="true" />
         <CheckBoxPreference
```

Comment:
I don't know a lot about LiveTV but as far as I know this checkbox should only be enabled when the selected player is external. Because the `dependency` attribute only works when referencing checkboxes this should be added in code.
You can add it in the `UserPreferencesFragment.kt` file.

Reply:
I’m not sure why this was marked as a dependency. This controls direct playing in the embedded players. I have no clue why this isn’t handled automatically like it is for normal videos. 🤷‍♂

Reply:
We can always make changes to that too in the future, I'll approve it for now :)